<?xml version="1.0"?>

<launch>

  <!-- Start unicycle dynamics -->
    <arg name="rate" default="10.0"/>
    <arg name="velocity_decay" default = "0.5"/>

    <arg name="pose" default="pose"/>
    <arg name="initial_pose" default="initial_pose"/>
    <arg name="cmd_vel" default="cmd_vel"/>


    <node pkg="unicycle_motion_control" type="unicycle_dynamics.py" name="unicycle_dynamics" output="screen">
        <param name="rate" value="$(arg rate)"/>
        <param name="velocity_decay" value="$(arg velocity_decay)"/>

        <remap from="pose" to="$(arg pose)"/>
        <remap from="initial_pose" to="$(arg initial_pose)"/>
        <remap from="cmd_vel" to="$(arg cmd_vel)"/>
    </node>


</launch>